### PR TITLE
ENH Default graphql abstractions to gql v5 style

### PR DESCRIPTION
**Pull request by GuySartorelli**
Following on from #1473, this replaces the GraphQL v3 defaults from the graphql schema react injector abstractions with GraphQL v4/5 defaults.

## Issue
- https://github.com/silverstripe/developer-docs/issues/62

### CODE REVIEW

File: client/src/lib/dependency-injection/tests/ApolloGraphqlScaffoldingContainer-test.js [hunk 1]
```diff
@@ -184,8 +184,8 @@ describe('Dynamic graphql injection', () => {
       singularName: 'Dino',
       pagination: false,
       params: {
-        // Declare that the "id" param is lowercase
-        id: 'ID!'
+        // Declare that the "id" param is uppercase
+        ID: 'ID!'
```
Comment (sabina-talipova) [inline client/src/lib/dependency-injection/tests/ApolloGraphqlScaffoldingContainer-test.js:188]: Small question, does it still work with lowercase `id`?

Comment (ChloeSartorelli) [reply client/src/lib/dependency-injection/tests/ApolloGraphqlScaffoldingContainer-test.js:188]: Yes - that's now the default in fact. See line 175 where another test is updated to account for that.

**Comment by sabina-talipova** [comment]
Looks good. Could you just rebase to 2.0, please.

**Comment by GuySartorelli** [comment]
@sabina Github says there are no conflicts - why do you want me to rebase this?

**Comment by sabina-talipova** [comment]
> @Sabina Github says there are no conflicts - why do you want me to rebase this?
We have couple new merged PRs. If you think it's not necessary, I'm happy to merge.

**Comment by GuySartorelli** [comment]
Yeah, it's not necessary. We only need to rebase if there are conflicts blocking the merge, or if the PR was originally targetting a different branch.

**Review by sabina-talipova — APPROVED** [review]

**Pull request merged**